optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -27,30 +38,53 @@ by using AI to adhere to hard and soft scheduling _constraints_, such as:
 
 Mathematically speaking, school timetabling is an _NP-hard_ problem.
 That means it is difficult to scale.
-Simply brute force iterating through all possible combinations takes millions of years
+Simply iterating through all possible combinations with brute force would take millions of years
 for a non-trivial dataset, even on a supercomputer.
-Luckily, AI constraint solvers such as OptaPlanner have advanced algorithms
+Fortunately, AI constraint solvers such as {PLANNER} have advanced algorithms
 that deliver a near-optimal solution in a reasonable amount of time.
+// Can we qualify what a reasonable amount of time is at this stage? Perhaps a range, or ", usually under 10 minutes."?
 
-== What you'll need
+.Prerequisites
 
 * https://www.oracle.com/java/technologies/javase-downloads.html[JDK 8 or later]
 * https://maven.apache.org/download.cgi[Maven 3.2+] or https://gradle.org/install/[Gradle4+]
-* An IDE, such as https://www.jetbrains.com/idea[IntelliJ IDEA], VSCode, Eclipse or NetBeans
+* An IDE, such as https://www.jetbrains.com/idea[IntelliJ IDEA], VSCode, Eclipse, or NetBeans
 
-== The build file and the dependencies
+== Generate the {PLANNER} Spring Boot project
 
-Use https://start.spring.io/[Spring Initializr] to generate an application
-with the following dependencies:
+Spring Initialzr is a web-based application that creates a Spring Boot structure based on a few user inputs. You can use Spring Initializr to easily generate a Maven or Gradle Spring Boot project that you can customize for {PLANNER}.
 
-* Spring Web (`spring-boot-starter-web`)
-* OptaPlanner (`optaplanner-spring-boot-starter`)
-** Currently `optaplanner-spring-boot-starter` isn't included in Spring Initializr yet.
-Add it manually in your build file.
 
-If you choose Maven, your `pom.xml` file has the following content:
-
-====
+.Procedure
+. Open Spring Initializr in a web browser:
++
+[source]
+----
+https://start.spring.io/
+----
+. Select a project, language, Spring Boot version, and enter project metadata.
+. Click *Add Dependencies* and select *Spring Web* to add the `spring-boot-starter-web` dependency.
+. Click *GENERATE*. Your project ZIP file downloads.
+. Extract the ZIP file and change directory to your Spring Boot project directory.
+. Add the {PLANNER} dependency (`optaplanner-spring-boot-starter`). Currently `optaplanner-spring-boot-starter` is not included in Spring Initializr so you must manually add it to your build file.
++
+* If you generated a Maven project, add the `optaplanner-spring-boot-starter` to your `pom.xml` file where `_VERSION_` is the verion of {PLANNER}:
```

Comment:
The backticks around `_VERSION_` shouldn't be there.

Actual output:
... file where `_VERSION_` is the version ...

Expected output
... file where _VERSION_ is the version ...

Alternatively, do not hardcode project version in the POM snippet below to avoid the `VERSION` placeholder.